pkgs/top-level/python-packages.nix

```
@@ -3150,6 +3150,8 @@ in {
 
   hyppo = callPackage ../development/python-modules/hyppo { };
 
+  i2c-tools = callPackage ../os-specific/linux/i2c-tools/python.nix { };
```

please move the file to `pkgs/development/python-modules/i2c-tools/default.nix`. To be co-located with other python modules
```suggestion
  i2c-tools = callPackage pkgs/development/python-modules/i2c-tools {
    inherit (pkgs) i2c-tools;
  };
```